x/upgrade/internal/keeper/utils.go

```diff
@@ -0,0 +1,18 @@
+package keeper
+
+func (k Keeper) ConvertIntArrayToInt64(blockHeight []int) []int64 {
```

Comment:
missing godoc comments

Reply:
also, we can move this file to `/types`. They don't depend on a `ctx sdk.Context` so there's no need to have them on the keeper.

Reply:
Agreed, will update